scripts/sign-registry.ts

```diff
@@ -66,8 +66,10 @@ async function main() {
 
   const registry = await fs.readFile(registryPath);
 
+  const hash = await sha256(new Uint8Array(registry));
```

Comment:
I have verified locally that this produces the same signature as the previous implementation